### DISCUSSION TITLE
Let the user change language & appearance in the first page of onboarding experience

### DISCUSSION THREAD

**Discussion by rafaeldjpbrochado** [iotaledger/firefly#1533] [Features / Enhancements]
## Description

The current onboarding experience is: 
- Choose Language -> Privacy Policy & Terms of Service -> Choose Appearance

I'm suggesting we make it:
- Choose Language & Appearance -> Privacy Policy & Terms of Service

Like in the original design:

![1-EZN9by-vs8DNfvwbQ7VzHQ](https://user-images.githubusercontent.com/39466455/111917537-dc227b80-8a56-11eb-9bab-5846e05a61a7.png)

## Motivation

This will let users read the Privacy Policy & Terms of Service using dark theme, if they so choose. Plus, it removes 1 page (and therefore an extra click) from the onboarding experience, which is nice.

## Are you planning to do it yourself in a pull request?

No.

**Comment by maxwellmattryan** [comment]
I think removing the extra click is a good motivation to do this! We could also use "Personalization" as the title / naming of the onboarding page?

**Reply by rafaeldjpbrochado** [reply]
Yeah, that sounds good!
